schema_utils_test.py

```diff
@@ -38,6 +38,13 @@
 SCHEMA_KEY_VALIDATORS = schema_utils.SCHEMA_KEY_VALIDATORS
 SCHEMA_KEY_DESCRIPTION = 'description'
 SCHEMA_KEY_UI_CONFIG = 'ui_config'
+# This key is used for 'type: custom' objects, as a way of indicating how
+# default ui_config values defined in objects.py should be replaced. The value
+# is a dictionary mapping the accessor of the object value to the ui_config.
+# For example, for SubtitledHtml (defined as a dict), to replace the ui_config
+# of the inner html schema, the accessor/key would be 'html'. Note that the
+# the actual ui_config is not replaced - the frontend needs to handle the
```

Comment:
Next PR: drop extra "the" at beginning of line

Reply:
Done